BitwardenShared/Core/Vault/Services/TOTP/TOTPKey.swift

```diff
@@ -70,13 +70,14 @@ enum TOTPKey: Equatable, Sendable {
     ///
     /// - Parameter key: A string representing the TOTP key.
     init?(_ key: String) {
-        if key.uppercased().isBase32 {
-            self = .base32(key: key)
-        } else if key.hasOTPAuthPrefix,
-                  let otpAuthModel = OTPAuthModel(otpAuthKey: key) {
+        let trimmedKey = key.replacingOccurrences(of: " ", with: "")
```

Comment:
🤔  Is there a more robust way to delete whitespace characters? Would it make sense to also just trim out everything that's not valid base32? That might be a UX thing, to error to indicate the user typed an invalid character in case they mistyped, but it might also be nice to just handle things without erroring?

Reply:
@fedemkr and I were going back and forth on this.  If we trim out everything that isnt base32 than that messes up the symbol padding suffixes for steam and otp.  I can go with the original plan and combine base32/unknown cases but Federico thought it would be best to just remove whitespaces.  Happy to do whatever you guys think is best, just not trying to break anything :)

Reply:
I think probably a bit more robust whitespace deletion would be sufficient, then. It would certainly improve things, and we can always circle back to consider how we handle the various sorts of TOTP keys we can get. In practice, I don't think people enter these very often, though, so it might not be worth the effort. Maybe something like `let trimmedKey = key.filter{ !$0.isWhitespace }`?

Reply:
okay makes sense to me!  i like it.

Reply:
If the SDK is removing any non-base32 characters, is it worth doing any filtering on our side?

Reply:
At most we'd want to find Steam and `otp://` keys, but I tend to agree with that. It might be worth double-checking what the SDK does in that regard, but if we don't have to sanitize the user's input like that, I'd rather just leave it.

Reply:
Right, I agree. It looks like the SDK is: https://github.com/bitwarden/sdk/blob/27a71d899284da50e9a08ca87a0068f991dafe44/crates/bitwarden-vault/src/totp.rs#L218-L224

Reply:
Then I think we should accept the user's input as-is and let the SDK handle it.

Reply:
i sync'd with David Severns on the android side of things and he confirmed they aren't doing any validation on the user input as well.